package.json

```diff
@@ -67,7 +67,6 @@
     "test:build-browser-container": "docker-compose up --build minimal-packager && docker-compose build --pull browser-maze-runner",
     "test:build-node-container": "docker-compose up --build minimal-packager && docker-compose build --pull node-maze-runner",
     "test:build-expo-android-container": "test/expo/scripts/build-android-locally.sh",
-    "test:build-expo-ios-container": "EXPO_RELEASE_CHANNEL=localtest test/expo/scripts/build-ios-locally.sh",
```

Comment:
This script is referenced in https://github.com/bugsnag/bugsnag-js/pull/1053/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R76, so we should amend that to prevent errors.

Reply:
Now removed and README updated.  I've raised an internal ticket to reinstate the ability to run Expo tests locally against an iOS device.